policyengine_us/parameters/gov/hhs/medicaid/eligibility/categories/senior_or_disabled/income/limit/couple.yaml

```diff
@@ -1,113 +1,112 @@
 description: Maximum income limit (of SSI countable income) for a couple to qualify for Medicaid under the optional senior or disabled pathway.
```

Comment:
could you adjust the description if this is is % of the federal poverty guidelines